modules/media/ajax/FileUpload.php

```diff
@@ -184,22 +186,34 @@ function uploadFile()
         'last_modified' => date("Y-m-d H:i:s"),
         'language_id'   => $language,
     ];
-
-    if (move_uploaded_file($_FILES["file"]["tmp_name"], $mediaPath . $fileName)) {
-        try {
+    
+      // upload to aws s3
+    if (getenv('AWS_ACCESS_KEY_ID') !== false) {
+    $s3_upload_status=false;
+$config      = \NDB_Config::singleton();
+$bucketName = $config->getSetting('AWS_S3_Default_Bucket');
```

Comment:
Might be worth making a new `AWS_S3_Bucket_Media` config? I think the S3_Default_Bucket is used for imaging and most likely projects would want media and imaging to go in different buckets.